flink-table-store-kafka/src/main/java/org/apache/flink/table/store/kafka/KafkaLogDeserializationSchema.java

```diff
@@ -85,22 +101,50 @@ public RowData deserialize(ConsumerRecord<byte[], byte[]> record) {
     }
 
     @Override
-    public void deserialize(ConsumerRecord<byte[], byte[]> record, Collector<RowData> out)
+    public void deserialize(
+            ConsumerRecord<byte[], byte[]> record, Collector<RowData> underCollector)
```

Comment:
what's the point of renaming `out` to `underCollector`?

Reply:
Avoid being misused, we should use new `collector`